new_component_todo-item.md

```diff
@@ -47,23 +49,23 @@ Now we need to pass a string, which is the item's title, where we use the compon
 </ul>
 ```
 
-The `itemTitle` here in square brackets is the same as declared as the component's `@Input`.
+O `itemTitle` entre colchetes é o mesmo que foi declarado como componente no `@Input`.
 
-We use property binding on an element we created ourselves! And now we can actually see and understand that property binding binds to actual property of the component. 
+Nós usamos a propriedade Binding em um elemento que nós mesmo criamos! E agora podemos realmente ver  e entender que a vinculação da propriedade Binding se liga a propriedade atual do componente.
```

Comment:
pls, add plural state for "mesmo" and the second phrase to better understanding.

Reply:
Thanks, corrected #063c991.